config/base/derive/src/documented.rs

```diff
@@ -47,17 +47,30 @@ fn impl_get_doc_recursive(docs: Vec<LitStr>, ast: &StructWithFields) -> proc_mac
         .zip(docs)
         .map(|(field, documentation)| {
             let inner_thing = field.has_inner;
```

Comment:
`option_thing` and `inner_thing` non-informative to me, in this case `has_inner`/`has_option` would be more descriptive.